Airports/EGBW/SMR/Labels.txt

```diff
@@ -0,0 +1,6 @@
+;Holding Points
+"A" N052.11.49.659 W001.36.58.012 standHold
+"B" N052.11.38.596 W001.36.56.887 standHold
+"C" N052.11.14.921 W001.37.07.607 standHold
+"D" N052.11.11.503 W001.37.01.790 standHold
+"E" N052.11.11.168 W001.36.54.183 standHold
```

Comment:
Move the labels slightly further from the stop bars - these are very close to the features on the SMR

Reply:
![image](https://user-images.githubusercontent.com/7039626/70160530-918abd80-16b2-11ea-97d6-3d5f4570c4c6.png)

Moved labels away from stop bars